src/go/rpk/pkg/cli/cluster/partitions/cancel.go

```diff
@@ -16,18 +16,19 @@ func newMovementCancelCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 		noConfirm bool
 	)
 	cmd := &cobra.Command{
-		Use:   "movement-cancel",
-		Short: "Cancel ongoing partition movements",
+		Use:     "move-cancel",
+		Short:   "Cancel ongoing partition movements",
+		Aliases: []string{"alter-cancel"},
```

Comment:
Making `movement-cancel` as an alias sounds simpler.

Reply:
@twmb should we mark `movement-cancel` deprecated? Isn't it a good idea to make `movement-cancel` as an alias?

Reply:
I leaned toward going with the second approach; Extract the whole command execution to a function and use it in both places.

The change is available in https://github.com/redpanda-data/redpanda/pull/13258/commits/4d84058e78fb59eacab384d23dcc0db98b5597cd

Reply:
Yeah I was thinking to not have a deprecation message, at lease for now, since movement-cancel is the currently documented way of things -- move-cancel is just shorter, and there's no reason we would want to show movement-cancel once we have move cancel.